test/items/item-model-test.js

```diff
@@ -104,6 +104,57 @@ describe('Item Model', function() {
     });
   });
 
+  describe('resort', function() {
+    beforeEach(function() {
+      this.server = sinon.fakeServer.create();
+      this.item = this.product.createItem({
+        number: 50,
+        product: {
+          id: 1
+        }
+      });
+      this.item.url = function() {
+        return '/products/1/items/50.json';
+      };
+    });
+
+    afterEach(function() {
+      this.server.restore();
+    });
+
+    it('returns a promise', function() {
+      this.server.respondWith(
+        'POST',
+        '/products/1/items/50/resort.json',
+        [
+          200,
+          { 'Content-Type': 'application/json' },
+          '{ "sort": 20000 }'
+        ]
+      );
+
+      var req = this.item.resort({ before: 2 });
+      this.server.respond();
+      return req;
```

Comment:
Feels like there should be an assert here, unless I don't know something about how mocha handles returns (likely)?